### PR TITLE
larger wysiwyg toolbar

### PR DESCRIPTION
**Pull request by nelsonpecora**
* 48px buttons
* material design icons
* [trello ticket](https://trello.com/c/Y22imh4W/1-text-selector-adjustments)

screen shot 2016-08-11 at 3 31 42 pm

***ZOOM AND ENHANCE!***

screen shot 2016-08-11 at 3 31 19 pm

### CODE REVIEW

File: behaviors/wysiwyg.js [hunk 1]
```diff
@@ -517,19 +517,27 @@ function initWysiwygBinder(enableKeyboardExtras) {
...
+        linkExtension = _.find(editor.extensions, findExtension('anchor')),
+        removeFormattingExtension = _.find(editor.extensions, findExtension('removeFormat'));
 
       // apply custom styling to buttons
+      if (boldExtension) {
+        boldExtension.button.innerHTML = `<img src="${site.get('assetPath')}/media/components/clay-kiln/bold.svg" />`;
+      }
       if (italicExtension) {
-        italicExtension.button.innerHTML = '<em>I</em>';
+        italicExtension.button.innerHTML = `<img src="${site.get('assetPath')}/media/components/clay-kiln/italics.svg" />`;
```
Comment (cruzanmo) [inline behaviors/wysiwyg.js]: Would be nice to de-couple the site and kiln. Could this be an option passed into kiln? Or have default images live in kiln?

File: behaviors/wysiwyg.scss [hunk 2]
```diff
@@ -54,10 +55,57 @@ $wysiwyg-dropshadow: 0 0 6px $input-shadow;
...
+//   filter: drop-shadow($wysiwyg-dropshadow);
+//   top: $wysiwyg-button-size;
+//   z-index: -1;
+// }
+
+.medium-toolbar-arrow-under:before,
 .medium-toolbar-arrow-under:after {
-  border-color: $wysiwyg-bgcolor transparent transparent transparent;
-  filter: drop-shadow($wysiwyg-dropshadow);
+	border: 1px solid transparent;
```
Comment (cruzanmo) [inline behaviors/wysiwyg.scss]: Looks like tabs?

File: behaviors/wysiwyg.js (hunk 1 above)
Comment (nelsonpecora) [inline behaviors/wysiwyg.js]: as kiln is a component that lives in a site, this'll always be relatively tightly coupled

File: behaviors/wysiwyg.scss (hunk 2 above)
Comment (nelsonpecora) [inline behaviors/wysiwyg.scss]: ?

File: behaviors/wysiwyg.js (hunk 1 above)
Comment (cruzanmo) [inline behaviors/wysiwyg.js]: True, I am fine with this since the images are living in the clay-kiln repo.

File: behaviors/wysiwyg.scss (hunk 2 above)
Comment (cruzanmo) [inline behaviors/wysiwyg.scss]: Meant that the indentations look inconsistent -- some are tabs and some are spaces.

**Comment by gloddy** [comment]
👍

**Comment by cruzanmo** [comment]
Nice, +1

**Pull request merged**